shortfin/tests/apps/llm/components/kvcache/page_pool_test.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+import logging
+from shortfin_apps.llm.components.kvcache.page_pool import PagePool, PagePoolConfig
+import shortfin as sf
+import shortfin.host
+import shortfin.array as sfnp
+import shortfin.amdgpu
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def setup_pool(generic_device):
```

Comment:
Is it possible to have a simulate multi-device using `generic_device`? That would be a nice test case to have

Reply:
I think it's probably possible!

We would just have to make it a list. Will create an issue for this.

Reply:
https://github.com/nod-ai/shark-ai/issues/599